### DISCUSSION TITLE
Cannot get the News Downloader config file runs

### DISCUSSION THREAD

**Discussion by toobluescientist** [koreader/koreader#12079] [Q&A] [answered]
Hi, I want to read RSS on KOReader on my Kindle. However, I cannot save entries into the feed_config.lua and hence, cannot download news. 

When I try to edit the _feed_config.lua_, for example,

`{"https://www.ehn.org/feeds/feed.rss", limit=2, download_full_article=true},`

when hitting the save button, the program would return: 
`Configuration invalid: Line 1: unexpected symbol near '{'`

Please help me to figure this out.

**Comment by benoit-pierre** [comment]
Did you keep the rest of the file?

**Reply by benoit-pierre** [reply, marked as answer]
At the minimum, it should look like this:
```lua
return {
    {"https://www.ehn.org/feeds/feed.rss", limit=2, download_full_article=true},
}

**Reply by toobluescientist** [reply]
You're a life saver. I have struggled with this for so long. Thank you so much!!!
